src/lib.rs

```diff
@@ -98,6 +98,12 @@
 //! incr!(Metric::Ringo; "play" => "drums");
 //! ```
 //!
+//! ## Note - Avoid high tag cardinality!
+//! It's important to avoid passing a large number of values for a given tag, as Datadog tracks each
+//! unique combination of tag values as a separate metric, which can significantly impact billing.
+//! For example, avoid passing things like user ID's, session ID's, request ID's, or other values that
```

Comment:
Why `user ID's` instead of `user IDs`? :thinking:

Reply:
Removed the apostrophes, I'm never really certain where to put them :laughing: